Cargo.lock

```diff
@@ -2109,6 +2109,7 @@ dependencies = [
  "its-consensus-slots",
  "its-primitives",
  "its-storage",
+ "its-test",
```

Comment:
I agree with this approach 👍 and yes, we don't want to add these to the regular `itp-test` crate, in order to keep the sidechain related stuff separate.